superset/connectors/sqla/models.py

```diff
@@ -1161,6 +1161,21 @@ def get_sqla_query(  # pylint: disable=too-many-arguments,too-many-locals,too-ma
             val = flt.get("val")
             op = flt["op"].upper()
             col_obj = columns_by_name.get(col)
+
+            if is_feature_enabled("ENABLE_TEMPLATE_REMOVE_FILTERS"):
+                # Assume the virtual table's jinja template does not process
+                # the filter.
+                filter_removed_by_vt = False
```

Comment:
It took me a few seconds to process this variable name. Could we just do something like `ignore_filter` and then make sure the comment + code explains what's going on?